api/proto/teleport/legacy/client/proto/authservice.proto

```diff
@@ -2715,4 +2741,25 @@ service AuthService {
 
   // DeleteAllSAMLIdPServiceProviders removes all SAML IdP service providers.
   rpc DeleteAllSAMLIdPServiceProviders(google.protobuf.Empty) returns (google.protobuf.Empty);
+
+  // CreatePlugin creates a new plugin instance.
+  rpc CreatePlugin(CreatePluginRequest) returns (google.protobuf.Empty);
```

Comment:
Can't we move these methods into a PluginService so we don't add more stuff into AuthService?

Reply:
Will try to move to a dedicated service a la `loginrule`.